modules/platform/nuxeo-coldstorage/src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
```

Comment:
Not sure you need this file at all.

Reply:
currently the defined test bundle is used in `ColdStorageFeature`